backend/tests/auth_api.test.ts

```diff
@@ -0,0 +1,271 @@
+import { test, afterEach, after, describe } from "node:test";
+import assert from "node:assert";
+import supertest from "supertest";
+import mongoose from "mongoose";
+import User from "../models/user";
+import app from "../app";
+import * as helper from "./test_helper";
```

Comment:
Could we avoid using namespace import `import * as ...`? I feel like we are repeating a lot of code, e.g `helper.abc`, without providing much context and sometimes it's even harder to read the code 🤔